### PR TITLE
Workshop Creation - WMS ID: 11252

### PR DESCRIPTION
**Pull request by heloisaescobar**
# Add here your WMS ID

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [ X] All filenames are lower case (including folders, images, files, etc.)
- [ X] Filenames are descriptive
- [X ] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [X ] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [ X] Image references in markdown contain an alternate text

### CODE REVIEW

**Review by denisemyrick — CHANGES_REQUESTED** [review]
Please verify all file names are in lowercase. Please remove LiveLabs-v3.zip

**Comment by rahul-tasker** [comment]
Please also update this branch, as it is out of date with the base branch.

**Comment by heloisaescobar** [comment]
updated name to live-labs-v3.zip

**Comment by heloisaescobar** [comment]
live-labs-v3.zip

File: oci-lakehouse-fast-track/live-labs-v3.zip
Comment (denisemyrick) [inline oci-lakehouse-fast-track/live-labs-v3.zip:1]: Hi, this .zip is still here. Is the .zip necessary for the lab? Is so it needs to be a PAR

**Review by denisemyrick — CHANGES_REQUESTED** [review]

**Comment by denisemyrick** [comment]
Thanks for removing the .zip. I see that the branch is still out of date though. Please update with base and we can approve the request.

**Comment by heloisaescobar** [comment]
Branch updated.

**Comment by denisemyrick** [comment]
Sorry, one more file with capitalized letters.
Screenshot 2024-01-04 at 9 34 27 AM

**Comment by heloisaescobar** [comment]
Rename 43-SQL-open.png to 43-sql-open.png

**Review by ramumg — APPROVED** [review]

**Pull request merged**